### PR TITLE
fix menubar toggle in mobile view

### PR DESCRIPTION
**Pull request by dhavalsavalia**
Steps took in order to fix issue:

- added `bootstrap.min.js` to make toggle work
- added `top-nav-collapse` class to `<nav>` to make it toggle-able
- also fixed PyCon logo which was previously not there

Here are screenshots:

## Desktop

screen shot 2018-06-13 at 7 16 30 pm

## Mobile

- Not toggled

screen shot 2018-06-13 at 7 17 37 pm

- Toggled

screen shot 2018-06-13 at 7 17 47 pm

### CODE REVIEW

File: convince-my-boss.html [hunk 1]
```diff
@@ -46,9 +46,11 @@
     <!-- <div id="preloader">
     <div id="load"></div>
   </div> -->
-    <nav class="navbar navbar-custom navbar-fixed-top" style="-webkit-box-shadow: 0 1px 30px rgba(0, 0, 0, .1);">
+    <nav class="navbar navbar-custom navbar-fixed-top top-nav-collapse" style="-webkit-box-shadow: 0 1px 30px rgba(0, 0, 0, .1);">
```
Comment (ananyo2012) [inline convince-my-boss.html]: I am not sure if this is required. We don't have this in other pages

Comment (ananyo2012) [inline convince-my-boss.html]: I am not sure if this class is required as it isn't present in other pages. Can you remove this and check ?

File: convince-my-boss.html [hunk 2]
```diff
@@ -46,9 +46,11 @@
...
         <div class="container">
-            <img src="img/banner-logo.png" class="logo" alt="banner-logo" id="logo" style="width: 150px;">
+            <a class="navbar-brand" rel="home" href="https://in.pycon.org/2018/" title="PyCon India 2018">
```
Comment (ananyo2012) [inline convince-my-boss.html]: Ok this looks like a bug. This has to be done in all pages. Can you open a separate issue and solve this in a separate PR ? Also the background color doesn't look good with the logo so just use white background.

File: convince-my-boss.html [hunk 3]
```diff
@@ -174,6 +176,7 @@ <h2 class="small-caps center">Convince My Boss</h2>
 
     <!--Core Javascript Files-->
     <script src="js/jquery.min.js"></script>
+    <script src="js/bootstrap.min.js"></script>
```
Comment (ananyo2012) [inline convince-my-boss.html]: Can you add all the javascript sources from the below list
```
<script src="js/jquery.min.js"></script>

  <script src="js/bootstrap.min.js"></script>

  <script src="js/jquery.easing.min.js"></script>

  <script src="js/jquery.scrollTo.js"></script>

   <script src="js/wow.min.js"></script>

    <script src="js/custom.js"></script>

   <script src="js/jtline.js"></script>

   <script src="js/canvas.js"></script>
```

File: convince-my-boss.html (hunk 2 above)
Comment (dhavalsavalia) [reply convince-my-boss.html]: Yeah, sure. Please review issue #135

File: convince-my-boss.html (hunk 3 above)
Comment (dhavalsavalia) [reply convince-my-boss.html]: Yes, I have added all js files. Just to let you know that `custom/css/reset.css` and `custom/css/default.css` are not there.

File: convince-my-boss.html (hunk 1 above)
Comment (dhavalsavalia) [reply convince-my-boss.html]: No, this class displays PyCon logo. I have tried to remove it PyCon logo disappears.

File: convince-my-boss.html (hunk 3 above)
Comment (dhavalsavalia) [reply convince-my-boss.html]: see 6401947

**Comment by dhavalsavalia** [comment]
Resolved merge conflict in 3fdc6fc

File: convince-my-boss.html (hunk 3 above)
Comment (ananyo2012) [reply convince-my-boss.html]: Uh they are already there https://github.com/pythonindia/inpycon2018/blob/master/convince-my-boss.html#L29-L30

**Comment by dhavalsavalia** [comment]
@ananyo2012 I have one confusion regarding items in navbar. Are they different across all pages or same? If they are same can you please confirm them?

**Comment by dhavalsavalia** [comment]
Okay. Even the js libraries that we added and css changes too?

**Comment by ananyo2012** [comment]
No only the changes from Line no 53-67

**Comment by dhavalsavalia** [comment]
@ananyo2012 sorry, but I don't get it

File: convince-my-boss.html [hunk 4]
```diff
@@ -36,35 +36,35 @@
...
-          <li><a href="index.html#speakers">Speakers</a></li>
-          <!-- <li><a href="index.html">Schedule</a></li> -->
-          <li><a href="index.html#sponsors">Sponsors</a></li>
-          <li><a href="index.html#tickets">Tickets</a></li>
-          <li><a href="https://in.pycon.org/blog/">Blog</a></li>
-          <li><a href="index.html#venue">Venue</a></li>
-          <li><a href="faq.html">FAQ</a></li>
-          <li><a href="coc.html">COC</a></li>
-        </ul>
+        <div class="collapse navbar-collapse" id="bs-example-navbar-collapse-1">
```
Comment (ananyo2012) [inline convince-my-boss.html]: @dhavalsavalia please remove the extra tabs in lines from 52 - 64. These are all unnecessary changes

File: convince-my-boss.html [hunk 5]
```diff
@@ -36,35 +36,35 @@
...
+          <ul class="nav nav-justified">
+            <li><a href="index.html#home">Home</a></li>
+            <li><a href="index.html#journey">Journey</a></li>
+            <!-- <li><a href="index.html">Speakers</a></li> -->
+            <!-- <li><a href="index.html">Schedule</a></li> -->
+            <li><a href="index.html#sponsors">Sponsors</a></li>
+            <li><a href="https://in.pycon.org/blog/">Blog</a></li>
```
Comment (ananyo2012) [inline convince-my-boss.html]: Don't interchange the links. It's not required.
